packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -59,6 +59,38 @@ def test_if_marathon_app_can_be_deployed_with_mesos_containerizer(cluster):
     cluster.marathon.deploy_test_app_and_check(app, test_uuid)
 
 
+def test_if_marathon_pods_can_be_deployed_with_mesos_containerizer(cluster):
+    """Marathon pods deployment integration test using the Mesos Containerizer
```

Comment:
nit: I would prefer that this test uses containers with health checks. If the execs totally fail, we won't know as the deployment will still consume and instance and appear OK. A simple exec of touch foo and health check of test -f foo to verify